crates/core/src/host/wasmtime/wasm_instance_env.rs

```diff
@@ -687,8 +685,9 @@ impl WasmInstanceEnv {
 
             // Insert the row into the DB.
             // This will return back the generated column values.
+            // TODO: Need to call `reducer_context` for catch the error?
             let ctx = env.reducer_context()?;
```

Comment:
Also this one

Reply:
Same here. `env` holds the `tx` which in turn holds the execution context. We're not explicitly passing it to `insert` anymore, so we can just remove this call.